detection-bot/README.md

```diff
@@ -0,0 +1,25 @@
+# Nethermind Bot Deployments Agent
+
+## Description
+
+This agent detects bots that are deployed by Nethermind
```

Comment:
```suggestion
This bot detects bots that are deployed by the Nethermind deployer address, [0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8](https://polygonscan.com/address/0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8)
```
Suggestion to refer to the exact Nethermind controlled address we're monitoring. Also, embedding the link to its appropriate block explorer page, in this case Polygonscan